Demo/Demo/Models/Order.swift

```diff
@@ -2,8 +2,8 @@ struct Order: Codable, Equatable {
     
     let id: String
     let status: String
-    var paymentSource: PaymentSource?
-    
+    let paymentSource: PaymentSource?
```

Comment:
This isn't related to this PR but at one point I was poking around at this file and noticed we had this one as a variable - should be a constant to match all other structs in this file